ksqldb-engine/src/main/java/io/confluent/ksql/internal/QueryStateMetricsReportingListener.java

```diff
@@ -104,35 +104,34 @@ public void onDeregister(final QueryMetadata query) {
     PerQueryListener(
         final Metrics metrics,
         final String groupPrefix,
-        final String queryApplicationId
+        final String queryId
     ) {
-      this(metrics, groupPrefix, queryApplicationId, CURRENT_TIME_MILLIS_TICKER);
+      this(metrics, groupPrefix, queryId, CURRENT_TIME_MILLIS_TICKER);
     }
 
     PerQueryListener(
         final Metrics metrics,
         final String groupPrefix,
-        final String queryApplicationId,
+        final String queryId,
         final Ticker ticker
     ) {
       Objects.requireNonNull(groupPrefix, "groupPrefix");
-      Objects.requireNonNull(queryApplicationId, "queryApplicationId");
+      Objects.requireNonNull(queryId, "queryId");
       this.metrics = Objects.requireNonNull(metrics, "metrics cannot be null.");
       this.ticker = Objects.requireNonNull(ticker, "ticker");
 
       this.stateMetricName = metrics.metricName(
           "query-status",
           groupPrefix + "ksql-queries",
           "The current status of the given query.",
-          Collections.singletonMap("status", queryApplicationId));
+          Collections.singletonMap("status", queryId));
 
       errorMetricName = metrics.metricName(
           "error-status",
           groupPrefix + "ksql-queries",
           "The current error status of the given query, if the state is in ERROR state",
```

Comment:
@rodesai what is this metric exactly? I don't think I see anything on the dashboard resembling it